### PR TITLE
Improve `react-big-calendar` UI & UX

### PR DESCRIPTION
**Pull request by diogogmatos**
Redesigned the whole `react-big-calendar` component to a more modern, appealing and user-friendly look. Most of the changes are grounded on the override of the rbc styling in `globals.css`, although some others were done via tweaks in the `<Calendar />` properties.

This included redesigning of the Week, Month and Schedule view. I took inspiration from TailwindUI's Calendar components.

**Preview:**

Month View
image

Week View
image

Schedule View
image

Toolbar
Screenshot_20230619_224114

**Comparisons (Before/After):**

Month View
image

Week View
image

Schedule View
image

### CODE REVIEW

**Comment by RuiL1904** [comment]
Clicking on "+2 more" is not working for me. Is it working for you, @diogogmatos?
image

**Comment by diogogmatos** [comment]
> Clicking on "+2 more" is not working for me. Is it working for you, @diogogmatos?
> 
> image
> 
> 

soo, the "+x more" button doesn't work because I think it would normally take you to the day view and I disabled the day view. I could try to make it redirect to the Week view instead.

**Comment by RuiL1904** [comment]
Make it redirect to the week view, then. @diogogmatos

**Comment by diogogmatos** [comment]
I can also make it redirect to day view but not display the day view button on the toolbar 👀

**Comment by RuiL1904** [comment]
As you want, tbh. Ideally, it would have an arrow side-by-side with the "2+ more" that, when clicked, would show the other two events of the given day (the current ones would disappear - just like it has two pages). But I don't know if that's possible regarding we are using rbc 😅
@diogogmatos

**Comment by diogogmatos** [comment]
yeah that's not possible ahaha, but ig i'll make it day view ;) ALSO, just made a discovery. apparently rbc considers an event "all day" when it surpasses midnight, showing it in that space at the top instead of where the hours show (in week or day view). since when we have an all day event we put the hour has "00:00 -> 23:59" it doesn't register as an all day event and just occupies the whole length of the view. however, we can make it "00:00 -> 00:00" and then it will show up as an all day event at the top! leaving the rest of the space to events that already have a defined interval of time and reducing overlaping of events! it looks like this:

image

image

@RuiL1904

**Comment by RuiL1904** [comment]
Yap, looks way nicer.
@diogogmatos

**Comment by diogogmatos** [comment]
@RuiL1904 fixed the "+x more" button, converted the events in events.json to support all-day events and updated the Helping Guide to reflect that change.

File: HELPING_GUIDE.md [hunk 1]
```diff
@@ -67,6 +67,9 @@ Where `<branch>` is the name of your branch.
 > For example: `221` (Bases de Dados).
 > Check out the `filterId` of each class by searching its name on `shifts.json` or checking the `id` parameter in `filters.json`.
 
+> **Note**
+> For all-day events, `start` and `end` should be equal and composed of: `<event-date> 00:00`. And for multiple-days events: `start: <event-start-date> 00:00` & `end: <event-end-date> 23:59`. Check out the example bellow.
```
Comment (RuiL1904) [inline HELPING_GUIDE.md:71]:
```suggestion
> For all-day events, `start` and `end` should be equal and composed of: `<event-date> 00:00`. And for multiple-days events: `start: <event-start-date> 00:00` & `end: <event-end-date> 23:59`. Check out the example below.
```

**Review by RuiL1904 — APPROVED** [review]
This is so much better 😍

**Pull request merged**